src/bellmanford.jl

```diff
@@ -105,3 +108,28 @@ function has_negative_edge_cycle{V, D}(
     edge_inspector = VectorEdgePropertyInspector{D}(edge_dists)
     has_negative_edge_cycle(graph, edge_inspector)
 end
+
+function enumerate_paths{V,D<:Number}(state::BellmanFordStates{V,D}, dest::Vector{V})
+    parents = state.parents
+    hasparent = state.hasparent
+    
+    num_dest = length(dest)
+    all_paths = Array(Vector{V},num_dest)
+    for i=1:num_dest
+        all_paths[i] = V[]
+        index = dest[i]
+        if hasparent[index] || parents[index] == index
+            while hasparent[index]
```

Comment:
BTW @sbromberger, thanks for all your recent work on this package, and I'm sorry I for one haven't been as efficient at reviewing and merging your work as I'd like to have been. It would be great if we could merge GraphCentrality.jl into Graphs.jl, but I think developing it in a separate repo for now is probably the best option.